src/main/java/de/oppermann/bastian/safetrade/listener/InventoryClickListener.java

```diff
@@ -181,10 +175,13 @@ private Inventory getClickedInventory(InventoryClickEvent event) {
         // https://hub.spigotmc.org/stash/projects/SPIGOT/repos/spigot/browse/Bukkit-Patches/0009-InventoryClickEvent-getClickedInventory.patch
         if (event.getRawSlot() < 0) {
             return null;
-        } else if (view.getTopInventory() != null && event.getRawSlot() < view.getTopInventory().getSize()) {
-            return view.getTopInventory();
         } else {
-            return view.getBottomInventory();
+            view.getTopInventory();
```

Comment:
This getter call seems unnecessary. Besides that: As you dropped Bukkit support by using `target.spigot().sendMessage(generateTextComponents());` anyway, we can just use `InventoryClickEvent#getClickedInventory` and drop the whole method.